src/style.css

```diff
@@ -1,3 +1,87 @@
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  background-color: #fff;
+  color: #00acdc;
+}
+
 body {
   margin: 0;
+  height: 60px;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.moyo_logo {
+  margin-top: 10px;
+  padding-left: 10px 0;
+  display: flex;
+}
```

Comment:
Not sure what this should do `padding-left: 10px 0;`, maybe you wanted `margin: 10px 0`